source/connect.txt

```diff
@@ -204,6 +204,21 @@ For ``tls`` connections:
 
         mongosh "mongodb://mongodb0.example.com:28015" --tls
 
+Connect to a Specific Database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to a specific database:
+
+- You can specify a database in the URL path when passing a connection
+  string. The database in the URL path is automatically selected unless
+  you specify a different database.
```

Comment:
This last sentence is confusing to me. Reading this, my question is "specify a different database where?". Can you explain (in a comment here) the idea you want to get across?